index.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href='http://fonts.googleapis.com/css?family=Roboto' rel='stylesheet' type='text/css'>
+    <link rel="stylesheet" href="./style.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <nav class="toolbar">
+        <div class="user-name">
+            <a href="#">Bisrat</a>
+        </div>
+        <div class="right-nav">
+            <a href="#"><img src="./img/line.png" alt="Email Icon"></a>
+        </div>
+    </nav>
+
+    <div class="headline">
+        <div class="full-name">
+            <h1>Bisrat M</h1>
+        </div>
+
+        <div class="carrier">
+            <h2>Software Developer</h2>
+        </div>
+    </div>
+
+    <div class="details">
+        <div class="content">
+            <p>I can help you build a product , feature or website Look through some of my work and experience! If you divke what you see and have a project you need coded,  don’t hestiate to contact me.</p>
+        </div>
+        <div class="break">
+            <hr>
+        </div>
+        <div class="experience">
+            <div class="language">
+                <strong>Language</strong>
+                <i class="lan-list">Javascript</i>
+                <i class="lan-list">Ruby</i>
+                <i class="lan-list">HTML</i>
+                <i class="lan-list">CSS</i>
+            </div>
+            <div class="frameworks">
+                <strong>Frameworks</strong>
+                <i class="framework-list">Bootstrap</i>
+                <i class="framework-list">Ruby on Rails</i>
+                <i class="framework-list">RSpec</i>
+                <i class="framework-list">CopyBara</i>
+                <i class="framework-list">Selenium</i>
+            </div>
+            <div class="skills">
+                <strong>Skills</strong>
+                <i class="skill-list">GitHub</i>
+                <i class="skill-list">GitLab</i>
+                <i class="skill-list">Terminal</i>
+                <i class="skill-list">Codekit</i>
+                <i class="skill-list">Codeopen</i>
+            </div>
```

Comment:
- [ ] **Required:** In these above sections, you are making a list of items, it is more _semantic_ to use [`ul` and `li`](https://stackoverflow.com/questions/13416866/why-should-i-use-ul-li-in-my-html) tags for this purpose as you have rightly done in the preceding section 👇. Try it out!